packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -278,8 +335,9 @@ export async function preTransformationUploadCode() {
             // if the user chose to skip unit tests, add the custom build command here
             transformZipManifest.customBuildCommand = transformByQState.getCustomBuildCommand()
             const zipCodeResult = await zipCode({
-                dependenciesFolder: transformByQState.getDependencyFolderInfo()!,
-                modulePath: transformByQState.getProjectPath(),
+                // dependenciesFolder will be undefined for SQL conversions since we don't compileProject
+                dependenciesFolder: transformByQState.getDependencyFolderInfo(),
+                projectPath: transformByQState.getProjectPath(),
```

Comment:
What's the difference between `modulePath` and `projectPath`?

Reply:
Nothing, just trying to be consistent as I call it a `project` elsewhere